src/fold/models/dummy.py

```diff
@@ -8,13 +8,47 @@
 
 class DummyClassifier(Model):
     """
-    A model that predicts a single class and probabilities.
+    A model that predicts a predefined class with predefined probabilities.
+
+    Parameters
+    ----------
+    predicted_value : Union[float, int]
+        The class to predict.
+    all_classes : List[int]
+        All possible classes.
+    predicted_probabilities : List[float]
+        The probabilities returned.
+
+    Examples
+    --------
+        >>> from fold.loop import train_backtest
+        >>> from fold.splitters import SlidingWindowSplitter
+        >>> from fold.models import DummyClassifier
+        >>> from fold.utils.tests import generate_sine_wave_data
+        >>> X, y  = generate_sine_wave_data()
+        >>> splitter = SlidingWindowSplitter(initial_train_window=0.5, step=0.2)
+        >>> pipeline = DummyClassifier(1, [0, 1], [0.5, 0.5])
+        >>> preds, trained_pipeline = train_backtest(pipeline, X, y, splitter)
+        >>> preds.head()
+                             predictions_DummyClassifier  ...  probabilities_DummyClassifier_1
+        2021-12-31 15:40:00                            1  ...                              0.5
+        2021-12-31 15:41:00                            1  ...                              0.5
+        2021-12-31 15:42:00                            1  ...                              0.5
+        2021-12-31 15:43:00                            1  ...                              0.5
+        2021-12-31 15:44:00                            1  ...                              0.5
+        <BLANKLINE>
```

Comment:
```suggestion
```

Reply:
unfortunately if I remove that then the tests fail